extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -404,8 +405,8 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
   /**
    * Returns the nearest ancestor element that is a suggested item.
-   * @param {?Element|?EventTarget} element
-   * @return {?Element|?EventTarget}
+   * @param {?Element} element
```

Comment:
Ah sorry, last one 😁`dev().assertElement` also returns a non-null value, so you can specify these params as `!Element`